### PR TITLE
first try sync tests

### PR DESCRIPTION
**Pull request by maggiemsft**
Trying to sync some of tests that were broke due to minigraph change.
It's supposed run against all platforms and I locally verified most of the tests, link link_flap, bgp_fact...

### CODE REVIEW

**Comment by lguohan** [comment]
can you change ansible/group_vars/str to ansible/group_vars/lab, and also change the inventory file to lab.

File: ansible/library/conn_graph_facts.py [hunk 1]
```diff
@@ -69,7 +69,6 @@
 '''
 
 LAB_CONNECTION_GRAPH_FILE = 'starlab_connection_graph.xml'
```
Comment (lguohan) [inline ansible/library/conn_graph_facts.py]: can you rename this file to lab_connection_graph.xml

**Comment by lguohan** [comment]
can you remove action_plugins and connection_plugins folder under ansible directory? they should not be used any more.

File: ansible/group_vars/sonic/vars [hunk 2]
```diff
@@ -16,14 +11,23 @@ cavium_hwskus: [ "AS7512", "XP-SIM" ]
 sensor_hwskus: [ "ACS-MSN2700", "Force10-S6000" ]
 
 ## Note:
-## Docker volumes should be list instead of dict. However, if we want to keep code DRY, we
+## Dokcer volumes should be list instead of dict. However, if we want to keep code DRY, we
```
Comment (stcheng) [inline ansible/group_vars/sonic/vars]: Docker

**Comment by maggiemsft** [comment]
too many update, tried rebase to a new branch with one push, all comments in this pull request are in the new branch.

**Pull request closed without merging**